testplan/web_ui/testing/src/Report/InteractiveReport.js

```diff
@@ -613,57 +614,63 @@ class InteractiveReport extends BaseReport {
-          updateEmptyDisplayFunc={noop}
-          updateTreeViewFunc={this.updateTreeView}
-          updateTagsDisplayFunc={noop}
-          updatePathDisplayFunc={this.updatePathDisplay}
-          updateTimeDisplayFunc={this.updateTimeDisplay}
-          extraButtons={[
-            <ReloadButton
-              key="reload-button"
-              reloading={this.state.reloading}
-              reloadCbk={this.reloadCode}
-            />,
-            <ResetButton
-              key="reset-button"
-              resetting={this.state.resetting}
-              resetStateCbk={this.resetReport}
-            />,
-            <AbortButton
-              key="abort-button"
-              aborting={this.state.aborting}
-              abortCbk={this.abortTestplan}
-            />,
-            <SaveButton key="save-button" />,
-          ]}
-        />
-        <Nav
-          interactive={true}
-          navListWidth={this.state.navWidth}
-          report={this.state.filteredReport.report}
-          selected={selectedEntries}
-          treeView={this.state.treeView}
-          filter={null}
-          displayEmpty={true}
-          displayTags={false}
-          displayTime={false}
-          // envCtrlCallback and handleClick are passed down to InteractiveNav
-          handleClick={this.handleClick}
-          envCtrlCallback={this.envCtrlCallback}
-          handleColumnResizing={this.handleColumnResizing}
-          url={this.props.match.path}
-        />
-        <AssertionContext.Provider value={this.state.assertionStatus}>
-          {centerPane}
-        </AssertionContext.Provider>
+        <ErrorBoundary>
+          <Toolbar
+            filterBoxWidth={this.state.navWidth}
+            filterText={this.state.filteredReport.filter.text}
+            status={reportStatus}
+            expandStatus={this.state.assertionStatus.globalExpand.status}
+            updateExpandStatusFunc={this.updateGlobalExpand}
+            handleNavFilter={this.handleNavFilter}
+            updateFilterFunc={noop}
+            updateEmptyDisplayFunc={noop}
+            updateTreeViewFunc={this.updateTreeView}
+            updateTagsDisplayFunc={noop}
+            updatePathDisplayFunc={this.updatePathDisplay}
+            updateTimeDisplayFunc={this.updateTimeDisplay}
+            extraButtons={[
+              <ReloadButton
+                key="reload-button"
+                reloading={this.state.reloading}
+                reloadCbk={this.reloadCode}
+              />,
+              <ResetButton
+                key="reset-button"
+                resetting={this.state.resetting}
+                resetStateCbk={this.resetReport}
+              />,
+              <AbortButton
+                key="abort-button"
+                aborting={this.state.aborting}
+                abortCbk={this.abortTestplan}
+              />,
+              <SaveButton key="save-button" />,
+            ]}
+          />
+        </ErrorBoundary>
+        <ErrorBoundary>
```

Comment:
if we do not have Nav I think it is enough to catch at top most, maybe the same for the toolbar.